From: Yoann Dubreuil <yoann.dubreuil@gmail.com>
Date: Sun, 18 Mar 2018 11:17:06 +0100
Subject: Full screen option

---
 flags.cpp      | 6 ++++++
 flags.h        | 1 +
 mainwindow.cpp | 3 +++
 3 files changed, 10 insertions(+)

diff --git a/flags.cpp b/flags.cpp
index 1acd42c..71fad8b 100644
--- a/flags.cpp
+++ b/flags.cpp
@@ -62,6 +62,7 @@ enum LongOption {
 	OPTION_10_BIT_OUTPUT,
 	OPTION_INPUT_YCBCR_INTERPRETATION,
         OPTION_ALSA_DELAY,
+	OPTION_FULL_SCREEN,
 };
 
 void usage(Program program)
@@ -159,6 +160,7 @@ void usage(Program program)
 		fprintf(stderr, "                                  Y'CbCr coefficient standard of card CARD (default auto)\n");
 		fprintf(stderr, "                                    auto is rec601 for SD, rec709 for HD, always limited\n");
 		fprintf(stderr, "                                    limited means standard 0-240/0-235 input range (for 8-bit)\n");
+		fprintf(stderr, "      --full-screen               Start in full screen\n");
 	}
 }
 
@@ -225,6 +227,7 @@ void parse_flags(Program program, int argc, char * const argv[])
 		{ "10-bit-input", no_argument, 0, OPTION_10_BIT_INPUT },
 		{ "10-bit-output", no_argument, 0, OPTION_10_BIT_OUTPUT },
 		{ "input-ycbcr-interpretation", required_argument, 0, OPTION_INPUT_YCBCR_INTERPRETATION },
+		{ "full-screen", no_argument, 0, OPTION_FULL_SCREEN },
 		{ 0, 0, 0, 0 }
 	};
 	vector<string> theme_dirs;
@@ -497,6 +500,9 @@ void parse_flags(Program program, int argc, char * const argv[])
 		case OPTION_ALSA_DELAY:
 			global_flags.alsa_delay = atoi(optarg);
 			break;
+		case OPTION_FULL_SCREEN:
+			global_flags.full_screen = true;
+			break;
 		case OPTION_HELP:
 			usage(program);
 			exit(0);
diff --git a/flags.h b/flags.h
index e1e7bf3..afdd72d 100644
--- a/flags.h
+++ b/flags.h
@@ -67,6 +67,7 @@ struct Flags {
 	int x264_bit_depth = 8;  // Not user-settable.
 	bool use_zerocopy = false;  // Not user-settable.
 	bool can_disable_srgb_decoder = false;  // Not user-settable.
+	bool full_screen = false;
 };
 extern Flags global_flags;
 
diff --git a/mainwindow.cpp b/mainwindow.cpp
index 7235e8e..470cd38 100644
--- a/mainwindow.cpp
+++ b/mainwindow.cpp
@@ -287,6 +287,9 @@ MainWindow::MainWindow()
 	}
 	midi_mapper.refresh_highlights();
 	midi_mapper.refresh_lights();
+	if (global_flags.full_screen) {
+		QMainWindow::showFullScreen();
+	}
 }
 
 void MainWindow::resizeEvent(QResizeEvent* event)
