From: "Steinar H. Gunderson" <sgunderson@bigfoot.com>
Date: Sat, 24 Mar 2018 17:49:52 +0100
Subject: Fix a buffer overrun when compiled without CEF.

---
 mixer.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mixer.h b/mixer.h
index effc2a3..eb128cf 100644
--- a/mixer.h
+++ b/mixer.h
@@ -442,7 +442,7 @@ private:
 	std::pair<std::string, std::string> get_channel_color_http(unsigned channel_idx);
 
 	HTTPD httpd;
-	unsigned num_cards, num_video_inputs, num_html_inputs;
+	unsigned num_cards, num_video_inputs, num_html_inputs = 0;
 
 	QSurface *mixer_surface, *h264_encoder_surface, *decklink_output_surface;
 	std::unique_ptr<movit::ResourcePool> resource_pool;
